ridgeback_description/urdf/ridgeback.urdf.xacro

```diff
@@ -296,6 +298,48 @@
     <origin xyz="0 0 ${deck_height}" rpy="0 0 0"/>
   </joint>
 
+  <!-- Gazebo Plugins -->
+  <gazebo>
+    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+      <robotNamespace>$(arg robot_namespace)</robotNamespace>
```

Comment:
Some suggest [this](https://answers.ros.org/question/241005/gazebo-topic-publication-issue-within-namespaced-group-tag/) which adds `-gazebo_namespace /xxx/gazebo` to the `spawn_model` node can avoid to define `<robotNamespace>` in the gazebo files. Perhaps this is something to try and see if we can minimize the delta.